.github/pull_request_template.md

```diff
@@ -17,7 +17,10 @@ See Story: [FSA22V2-ISSUE](https://sparkbox.atlassian.net/browse/FSA22V2-ISSUE)
 
 1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
 2. Pull down all related branches.
-3. Confirm all tests pass.
+3. In terminal: npm run dev.
+4. Check localhost: 3000 for changes.
```

Comment:
Could you remove the space between localhost: and 3000 on this line?

Reply:
good eye! 👀